lib/Value/AnswerChecker.pm

```diff
@@ -590,6 +590,7 @@ sub ans_collect {
 	$entry = $ans->{original_student_ans};
 	$ans->{student_formula} = $ans->{student_value} = undef unless $entry =~ m/\S/;
       }
+      $ans->{typeError} = 0;
```

Comment:
`entryCheck` seems to use the value of `typeError` from `$result`, so it is not clear to me whether the line  `$ans->{typeError} = 0;` is necessary.
Could this have adverse consequences for something else which depends on it?

Reply:
I have removed the line that sets it (I was mistakenly thinking they used the same answer hash, but they aren't the same, as you point out).